김현경/피보나치_수.py

```diff
@@ -0,0 +1,19 @@
+def solution(n):
+    a, b = 0, 1 # a는 0번째 피보나치 수, b는 1번째 피보나치 수
+    
+    for _ in range(2, n+1):
+        a, b = b, (a+b)% 1234567
```

Comment:
```suggestion
        a, b = b, (a+b) % 1234567
```
% 앞에 띄어쓰기 넣 좋을 것 같아요! ㅎㅎ

```suggestion
        a, b = b, (a+b)%1234567
```
아니면 +부분도 다 붙이니까 아에 붙여버리기?~?
(저는 다 띄우는 거 선호하긴 합니당 ㅎㅎ)

Reply:
앗 실수했네용 수정했습니다~~!

Reply:
오 suggestion은 처음봤네요 신기신기